srcpkgs/iamb/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'iamb'
+pkgname=iamb
+version=0.0.9
+revision=1
+build_style=cargo
```

Comment:
Upstream recommends linking against system-wide packages. I haven't tested this myself though, nor do I know if it's a good idea or not
(likewise, no idea about the lto thing either)
https://github.com/ulyssa/iamb/blob/main/PACKAGING.md#linking-against-system-packages
```suggestion
build_style=cargo
configure_args="--no-default-features --features=native-tls"
```

Reply:
That probably makes sense, I'll see if I can make that work (we might need some additional make depends then).

Reply:
Seems to work :+1: